spec/factories_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+describe 'factories' do
+  FactoryBot.factories.each do |factory|
+    describe "#{factory.name} factory" do
+      it 'does not raise error when built' do
+        expect { build(factory.name) }.not_to raise_error
+      end
+
+      it 'does not raise error when created' do
+        expect { create(factory.name) }.not_to raise_error
+      end
+
```

Comment:
There's also this which might be fun https://github.com/djezzzl/factory_trace

Reply:
Started it in https://github.com/department-of-veterans-affairs/vets-api/pull/3345